PrimeNumberTheoremAnd/ResidueCalcOnRectangles.lean

```diff
@@ -228,6 +229,269 @@ that the inner square is strictly contained in the big rectangle.)
 \end{proof}
 %%-/
 
+theorem ResidueTheoremAtOrigin_aux1a1 (f : ℝ → ℝ) : 
+  ∫ (x : ℝ) in (-1)..1, (f x : ℂ) = ∫ (x : ℝ) in (-1)..1, (f x : ℝ) := by
+  exact intervalIntegral.integral_ofReal
```

Comment:
If it's a one-line exact, then it probably doesn't need to be a standalone theorem?

Reply:
nice observation!